backend/graph/api/schema.resolvers.go

```diff
@@ -48,6 +48,44 @@ func (r *mutationRootResolver) SetDevicePushToken(ctx context.Context, token str
 	}, nil
 }
 
+// EpisodeProgress is the resolver for the episodeProgress field.
+func (r *mutationRootResolver) EpisodeProgress(ctx context.Context, id string, progress *int, duration *int) (*model.Episode, error) {
```

Comment:
Maybe the resolvers for mutations should include a verb, like `SetEpisodeProgress`?